src/ui.js

```diff
@@ -0,0 +1,157 @@
+// UI Class: Handles UI Tasks
+
+export default class UI {
+  static addApp(todos) {
+    this.addTitle();
+    this.addForm();
+    this.addEmptyUL();
+    // const todos = Store.getTasks();
+    this.addTasksUI(todos);
+    this.addbottombtn();
+  }
+
+  static addTitle() {
+    const appDiv = document.querySelector('#appDiv');
+
+    const div4title = document.createElement('div');
+    div4title.className = 'd-flex justify-content-start align-items-center border-bottom border-2 px-2 appItem';
+
+    const title = document.createElement('p');
+    title.className = 'fs-5 m-0';
+    title.innerText = "Today's To Do";
+
+    const icon = document.createElement('i');
+    icon.className = 'fas fa-sync-alt ms-auto p-2';
+
+    div4title.appendChild(title);
+    div4title.appendChild(icon);
+
+    appDiv.appendChild(div4title);
+  }
+
+  static addForm() {
+    const appDiv = document.querySelector('#appDiv');
+
+    const div4form = document.createElement('div');
+    const form = document.createElement('form');
+    form.className = 'd-flex justify-content-start align-items-center border-bottom border-2 px-2 appItem';
+    form.action = 'submit';
+    form.id = 'addTaskForm';
+
+    const input = document.createElement('input');
+    input.type = 'text';
+    input.id = 'taskDesc';
+    input.placeholder = 'Add to your list...';
+    input.className = 'form-control border-0 fst-italic p-0';
+
+    const icon = document.createElement('i');
+    icon.className = 'fas fa-sign-in-alt ms-auto p-2';
+
+    const aSubmit = document.createElement('a');
+    aSubmit.setAttribute('id', 'clickEnterIcon');
+
+    form.appendChild(input);
+    aSubmit.appendChild(icon);
+    form.appendChild(aSubmit);
+    div4form.appendChild(form);
+
+    appDiv.appendChild(div4form);
+  }
+
+  static addEmptyUL() {
+    const appDiv = document.querySelector('#appDiv');
+
+    const div4list = document.createElement('div');
+    const list = document.createElement('ul');
+    list.id = 'task-list';
+    list.className = 'p-0 m-0';
+    div4list.appendChild(list);
+
+    appDiv.appendChild(div4list);
+  }
+
+  static addEmptyToDoMessage() {
+    const list = document.querySelector('#task-list');
+
+    const item = document.createElement('li'); // creates list item
+    item.className = 'd-flex justify-content-center align-items-center border-bottom border-2 px-2 appItem';
+
+    const p = document.createElement('p');
+    p.className = 'm-0 p-0 noToDos';
+    p.innerHTML = "No to-do's right now";
+
+    item.appendChild(p);
+    list.appendChild(item);
+  }
+
+  static addTasksUI(tasks) {
+    // Iterates over array tasks to populate HTML list
+    if (tasks.length === 0) {
+      this.addEmptyToDoMessage();
+    } else {
+      tasks.forEach((task) => this.addTaskToList(task));
+    }
+  }
```

Comment:
- [ ] The dynamic rendering is not yet correct. If you change the index of the first task in your task array to `1` and the second one to `0` you still have the same render as if not changed. As the requirements say `On page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the index values for each task.` Please fix that!